rholang/src/main/scala/coop/rchain/rholang/interpreter/Interpreter.scala

```diff
@@ -52,9 +52,9 @@ object Interpreter {
       ): F[EvaluateResult] = {
         implicit val rand: Blake2b512Random = Blake2b512Random(128)
         for {
-          checkpoint <- runtime.space.createCheckpoint()
+          checkpoint <- runtime.space.createSoftCheckpoint()
```

Comment:
@ArturGajowy @JosDenmark Why are we creating a checkpoint in the interpreter?

Reply:
To be able to revert in case of errors 2 lines below